commit c914896aec1b33602e6168bf0aa7678a2e80bd89 (refs/changes/62/2562/3)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-11-24T13:53:04-08:00 (1 year, 10 months ago)
    
    DOCKER-1102 docker pull by manifest digest fails with an InternalError
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 0f9080b..50c277b 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -8,6 +8,10 @@
 
 ## not yet released
 
+## 10.4.1
+
+- DOCKER-1102 docker pull by manifest digest fails with an InternalError
+
 ## 10.4.0
 
 - Integration of [RFD 26](https://github.com/joyent/rfd/tree/master/rfd/0026)'s
diff --git a/lib/imgapi.js b/lib/imgapi.js
index 8d362a1..bf9cead 100644
--- a/lib/imgapi.js
+++ b/lib/imgapi.js
@@ -1127,8 +1127,8 @@ IMGAPI.prototype.adminImportDockerImage = function adminImportDockerImage(
     assert.optionalString(options.tag, 'options.tag');
     assert.optionalString(options.digest, 'options.digest');
     if (!options.tag && !options.digest) {
-        throw new assert.AssertionError(
-            'one of "options.tag" or "options.digest" is required');
+        throw new assert.AssertionError({ message:
+            'one of "options.tag" or "options.digest" is required' });
     }
     assert.optionalString(options.regAuth, 'options.regAuth');
     assert.optionalString(options.regConfig, 'options.regConfig');
diff --git a/package.json b/package.json
index ec4f657..48c9fdf 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdc-clients",
   "description": "node.js client libraries for Triton core REST APIs.",
-  "version": "10.4.0",
+  "version": "10.4.1",
   "homepage": "http://www.joyent.com",
   "repository": {
     "type": "git",
